internal/api/openapi.yaml

```
@@ -187,6 +187,146 @@ components:
+        data:
+          type: string
+          description: Base64-encoded ciphertext (nonce + AES-GCM payload) of the inner Payload struct.
+        summary:
+          type: object
+          description: |
+            Plaintext per-section row counts for sanity-checking without
+            decryption. Optional on import: legacy envelopes pre-1.2 do not
+            carry libraries/api_tokens/users counters, so all section fields
+            are optional here. The export-side schema is the strict counterpart.
+          properties:
+            settings: { type: integer }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)